utils/wasm-gen/src/generator/syscalls/imports.rs

```diff
@@ -280,22 +280,21 @@ impl<'a, 'b> SysCallsImportsGenerator<'a, 'b> {
                 .into();
             initial_mem_size.memory_size()
         };
+
         // subtract to be sure we are in memory boundaries.
-        let reserve_gas_result_ptr = memory_size_in_bytes.saturating_sub(100) as i32;
-        let rid_pid_value_ptr = reserve_gas_result_ptr + mem::size_of::<Length>() as i32;
-        let pid_value_ptr = reserve_gas_result_ptr + mem::size_of::<ErrorWithHash>() as i32;
-        let reservation_send_result_ptr = pid_value_ptr + mem::size_of::<HashWithValue>() as i32;
+        let pid_value_ptr = memory_size_in_bytes.saturating_sub(100) as i32;
+        let rid_pid_value_ptr = pid_value_ptr + mem::size_of::<HashWithValue>() as i32;
 
         let func_instructions = Instructions::new(vec![
             // Amount of gas to reserve
             Instruction::GetLocal(4),
             // Duration of the reservation
             Instruction::GetLocal(5),
             // Pointer to the LengthWithHash struct
-            Instruction::I32Const(reserve_gas_result_ptr),
+            Instruction::GetLocal(6),
```

Comment:
Is local variable with index 6 defined? I see indexes from 0 to 5 in signature
https://github.com/gear-tech/gear/blob/8b54acaf652807d4e93cffc9b2b719751a939483/utils/wasm-gen/src/generator/syscalls.rs#L78-L85

Reply:
https://github.com/gear-tech/gear/blob/md-fuzzer-syscall-results/utils/wasm-gen/src/generator/syscalls.rs#L79C6-L95C6